src/components/ContactDetails.vue

```diff
@@ -83,13 +83,16 @@
 
 				<!-- properties iteration -->
 				<!-- using contact.key in the key and index as key to avoid conflicts between similar data and exact key -->
-				<contact-details-property v-for="(property, index) in sortedProperties" :key="index+contact.key" :index="index"
+				<contact-property v-for="(property, index) in sortedProperties" :key="index+contact.key" :index="index"
```

Comment:
We had actually been talking about whether to use both "contact" and "details" or not when naming the component we created for the avatar management issue.

Reply:
Nice! What did you settle for? :)

Reply:
ContactAvatar, like you. My concern is just that we also have the list avatar to think of.